src/System.IO.FileSystem.Watcher/tests/System.IO.FileSystem.Watcher.Tests.builds

```diff
@@ -4,30 +4,28 @@
   <ItemGroup>
     <Project Include="System.IO.FileSystem.Watcher.Tests.csproj">
       <OSGroup>Linux</OSGroup>
+      <TargetGroup>netstandard1.3</TargetGroup>
+      <TestTFMs>netcoreapp1.0</TestTFMs>
     </Project>
     <Project Include="System.IO.FileSystem.Watcher.Tests.csproj">
       <OSGroup>OSX</OSGroup>
+      <TargetGroup>netstandard1.3</TargetGroup>
+      <TestTFMs>netcoreapp1.0</TestTFMs>
     </Project>
     <Project Include="System.IO.FileSystem.Watcher.Tests.csproj">
       <TestTFMs>netcoreapp1.0;net46</TestTFMs>
       <OSGroup>Windows_NT</OSGroup>
+      <TargetGroup>netstandard1.3</TargetGroup>
     </Project>
     <Project Include="System.IO.FileSystem.Watcher.Tests.csproj">
-      <TargetGroup>netstandard1.7</TargetGroup>
```

Comment:
It seems like it would have been better to just swap ns17 with the older builds so the diff would be logically ordered with the default builds at the top.